File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/IFS0001U00GrdMasterInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
    public class IFS0001U00GrdMasterInfo
    {
        private String _codeType;
        private String _acctType;
        private String _codeTypeName;
        private String _remark;
        private String _rowState;

        public String CodeType
        {
            get { return this._codeType; }
            set { this._codeType = value; }
        }

        public String AcctType
        {
            get { return this._acctType; }
            set { this._acctType = value; }
        }

        public String CodeTypeName
        {
            get { return this._codeTypeName; }
            set { this._codeTypeName = value; }
        }

        public String Remark
        {
            get { return this._remark; }
            set { this._remark = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public IFS0001U00GrdMasterInfo() { }

        public IFS0001U00GrdMasterInfo(String codeType, String acctType, String codeTypeName, String remark, String rowState)
        {
            this._codeType = codeType;
            this._acctType = acctType;
            this._codeTypeName = codeTypeName;
            this._remark = remark;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/IFS0001U00GrdMasterInfo.java
package nta.med.data.model.ihis.adma;

public class IFS0001U00GrdMasterInfo {
	private String codeType;
	private String codeTypeName;
	private String remark;
	public IFS0001U00GrdMasterInfo(String codeType, String codeTypeName,
			String remark) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
		this.remark = remark;
	}
	public String getCodeType() {
		return codeType;
	}
	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}
	public String getCodeTypeName() {
		return codeTypeName;
	}
	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
}
